java-koom混淆

混淆处理下啊

Caused by: java.lang.IllegalAccessException: void com.kwai.koom.javaoom.monitor.OOMMonitor.<init>() is not accessible from java.lang.Class<com.kwai.koom.base.MonitorManager>
        at java.lang.Class.newInstance(Native Method)
        at com.kwai.koom.base.MonitorManager.a(MonitorManager.kt:60)

--- a/koom-monitor-base/consumer-rules.pro
+++ b/koom-monitor-base/consumer-rules.pro
@@ -1,0 +1,1 @@
+-keep class * extends com.kwai.koom.base.Monitor { *; }

